### ISSUE TITLE
wither storm switches phases after rejoining

### ISSUE THREAD

**Issue by ROBOY6321** [nonamecrackers2/crackers-wither-storm-mod#1234]
***Please check the guidelines before submitting:*** https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/217

**Minecraft version**
1.19.4

**Mod version**
last version

**Describe the bug**
basically when wither storm is at small phase 5 and you rejoin it becomes bigger same thing with phase 6

**To Reproduce**
1. get wither storm to phase 5 or 6
2. rejoin the world
3. wat

**Crash Reports / Logs**
Crash reports / logs hosted on pastebin, gist, etc.

**Is OptiFine installed?**

no

**Screenshots**

**Comment by IndigoAmethyst** [comment]
Do you mean phase 5 and 5.5, because phase 6 is the phase after the F-Bomb?

**Comment by ROBOY6321** [comment]
> Do you mean phase 5 and 5.5, because phase 6 is the phase after the F-Bomb?

no

**Comment by ROBOY6321** [comment]
if you rejoin when wither storm is phase 5 it will become phase 5.5 and if you rejoin when wither storm phase 6 it becomes phase 7

**Comment by nonamecrackers2** [comment]
What other mods do you have installed?

**Comment by nonamecrackers2** [comment]
Closing until more information is given

**Comment by IndigoAmethyst** [comment]
The most simular issue I had to it was a time where I was being "Chased" by a Phase 3 Wither Storm. By chased I mean it wasn't right above me and constantly attacking, I mean I was sprint jumping for about an hour and it was able to stay in bossbar range until I got a dolphin or I got lucky and it got distracted with a dozen mobs. Anyway it was chasing me for so long that I had to die by a bed several times to stop Wither Sickness from killing me (Yes I used to cheese it like that), so I decided to end the stream and log off.

A few days later, I joined the World only to see a Phase 5 Wither Storm (Not to be confused with 5.5), completely skipping Phase 4.

It's worth noting that this was quite a while ago in and old version, and I had changed dozens of World Configs including evolution, I just thought to bring this up as it sounds similar
